core/src/main/java/org/apache/iceberg/inmemory/InMemoryFileIO.java

```diff
@@ -28,7 +28,7 @@
 
 public class InMemoryFileIO implements FileIO {
 
-  private final Map<String, byte[]> inMemoryFiles = Maps.newConcurrentMap();
+  private static final Map<String, byte[]> inMemoryFiles = Maps.newConcurrentMap();
```

Comment:
I am good with this, seems better to be a static variable so it's shared within the same JVM. Curious what others think.

Reply:
+ 1 for static and for using proper casing of variable.